pkg/operator/operator.go

```diff
@@ -127,6 +129,7 @@ func NewOperator() (context.Context, *Operator) {
 
 	// Client
 	kubernetesInterface := kubernetes.NewForConfigOrDie(config)
+	apiextensionsInterface := clientset.NewForConfigOrDie(config)
```

Comment:
You shouldn't need to create a new interface just for the apiextensions, you could probably re-use the `kubernetesInterface` here

Reply:
Removed.